JortPob/ESM/Content.cs

```diff
@@ -144,7 +143,149 @@ public Travel(JsonNode json) : base(json)
             }
         }
 
-        public NpcContent(Cell cell, JsonNode json, Record record) : base(cell, json, record)
+        public class Stats
+        {
+            public enum Tier { Novice = 0, Apprentice = 25, Journeyman = 50, Expert = 75, Master = 100 }
+            public enum Skill { Acrobatics, Alchemy, Alteration, Armorer, Athletics, Axe, Block, BluntWeapon, Conjuration, Destruction, Enchant, HandToHand, HeavyArmor, Illusion, LightArmor, LongBlade, Marksman, MediumArmor, Mercantile, Mysticism, Restoration, Security, ShortBlade, Sneak, Spear, Speechcraft, Unarmored };
+            public enum Attribute { Strength, Intelligence, Willpower, Agility, Speed, Endurance, Personality, Luck };
+
+            private readonly Dictionary<Skill, int> skills;
+            private readonly Dictionary<Attribute, int> attributes;
+
+            /* Defined stats constructor */
+            public Stats(JsonNode json)
+            {
+                attributes = new();
+                skills = new();
+
+                JsonArray jsonAttributes = json["attributes"].AsArray();
+                JsonArray jsonSkills = json["skills"].AsArray();
+
+                int i = 0;
+                foreach (Attribute attribute in Enum.GetValues(typeof(Attribute)))
+                {
+                    attributes.Add(attribute, jsonAttributes[i++].GetValue<int>());
+                }
+
+                i = 0;
+                foreach (Skill skill in Enum.GetValues(typeof(Skill)))
+                {
+                    skills.Add(skill, jsonSkills[i++].GetValue<int>());
+                }
+            }
+
+            /* Autocalculated stats constructor */
+            public Stats(Sex sex, RaceInfo raceInfo, JobInfo jobInfo, int level)
+            {
+                attributes = new();
+                skills = new();
+
+                foreach (Attribute attribute in Enum.GetValues(typeof(Attribute)))
+                {
+                    float baseVal = raceInfo.Get(sex, attribute);  // base racial value for attribute
+                    float bonus = 0;
+                    if(jobInfo.Has(attribute)) { baseVal += 10f; }
+                    foreach(Skill skill in Enum.GetValues(typeof(Skill)))
+                    {
+                        if(attribute == GetParent(skill))
```

Comment:
seems like being able to just directly lookup `GetAssociatedSkills` for an attribute would be helpful, instead of needing to search across all skills for every attribute